release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -80,6 +80,11 @@ This release introduces the following updates to the *Developer* perspective of
 [id="ocp-4-16-openshift-cli"]
 === OpenShift CLI (oc)
 
+[id="ocp-4-16-oc-adm-upgrade-status-tp"]
+==== Introducing the `oc adm upgrade status` command (Technology Preview)
+
+Previously, users could run the `oc adm upgrade` command to retrieve limited information about the cluster update status. With this release, there is now a technology preview `oc adm upgrade status` command in addition to `oc adm upgrade`, which decouples the status information from `oc adm upgrade` and complements the UI by providing customers with specific information regarding their cluster update, including the status of the control plane and worker node updates.
```

Comment:
```suggestion
Previously, users could run the `oc adm upgrade` command to retrieve limited information about the cluster update status. With this release, a technology preview `oc adm upgrade status` command is available in addition to `oc adm upgrade`. The new command removes the status information from `oc adm upgrade`. Specific information regarding your cluster update can now be obtained with the `oc adm upgrade status` command and includes the status of control plane and worker node updates.
```
"complementing the UI ..." this phrase seems meaningless to me; what does it mean? it's the same info as available in the console, er?